src/api/client.ts

```diff
@@ -2,13 +2,10 @@ import axios, { AxiosRequestConfig, AxiosInstance } from 'axios';
 
 import { config } from './config';
 
-export const createApiInstance = () =>
-  axios.create({
-    baseURL: config.baseURL,
-    headers: { Accept: 'application/json' },
-  });
-
-const api = createApiInstance();
+const api = axios.create({
```

Comment:
To be honest, I completely don't understand what's this module does. I know what should be the purpose of it, but in its current form - it doesn't make sense at all.

It's supposed to be a template for API clients but so far it's not customizable (i.e. `baseUrl` should be provided as a parameter to `createApiInstance`) and methods of `setRequestInterceptor`, `setResponseInterceptor` do literally nothing.

If we don't have an idea how to use `setRequestInterceptor`, `setResponseInterceptor` I think it'd be better to delete the module at all and move the logic `createApiInstance()`/`axios.create()` to `src/api/emailApi.ts`